dbms/sqlquery/mysql/src/cloud/benchflow/monitors/querymysql.go

```diff
@@ -39,19 +40,30 @@ func queryHandler(w http.ResponseWriter, r *http.Request) {
         }
         if name == value {
         	if(method == "equal") {
-        		fmt.Fprintf(w, "Row "+strconv.Itoa(rowI)+" matches "+value+" \n")
+        		//fmt.Fprintf(w, "Row "+strconv.Itoa(rowI)+" matches "+value+" \n")
+        		response = Response{true}
         		}
         	}
         if name != value {
         	if(method == "nequal") {
-        		fmt.Fprintf(w, "Row "+strconv.Itoa(rowI)+" doesn't match "+value+" \n")
+        		//fmt.Fprintf(w, "Row "+strconv.Itoa(rowI)+" doesn't match "+value+" \n")
+        		response = Response{true}
         		}
         	}
     }
     db.Close()
+    
+    js, err := json.Marshal(response)
```

Comment:
@VincenzoFerme Depends on the response format. Better keep this for each monitor for now